### DISCUSSION TITLE
Save ItemStack to file

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#970] [Help]
Hi,
How can I save an ItemStack in a file (Im using json as file type)?
Do I have to store the whole meta/nbt? or is there an easier way?
~Jochem

**Comment by TheMode** [comment]
`ItemStack#toItemNBT`
Alternatively `Tag.ItemStack` and then serialize the tag handler to nbt.

**Reply by DevBlokJochem** [reply]
How can I get the itemstack as a Tag? I know how to get the itemstack from  a Tag.
